src/wasm/wasm-ir-builder.cpp

```diff
@@ -576,6 +576,14 @@ Result<> IRBuilder::visitStringEncode(StringEncode* curr) {
   WASM_UNREACHABLE("unexpected op");
 }
 
+Result<> IRBuilder::visitContNew(ContNew* curr) {
+  auto func = pop();
+  CHECK_ERR(func);
+  curr->func = *func;
+
+  return Ok{};
+}
```

Comment:
The default `visitExpression` implementation will already do exactly this, so I don't think we need to implement `visitConstNew` here at all. If what we popped depended on the `contType` at all, that would be different.

Reply:
oh, nice!